google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -105,5 +112,10 @@ def _format_violation(violation, resource_name):
     Returns:
         Formatted violations
     """
+
     formatted_output = vm.VIOLATION_MAP[resource_name](violation)
     return formatted_output
+
```

Comment:
Would it be better to make this a class level attribute? If not, move it up before the class so it's easier to see before a reader starts to understand the class itself.